### ISSUE TITLE
Radio Button Selection not Visible

### ISSUE THREAD

**Issue by ninja511** [carbon-design-system/carbon#1133]
Moving from Carbon-components-angular repo:
https://github.com/IBM/carbon-components-angular/issues/88

Detailed description
Describe in detail the issue you're having.
During our AVT 2, discovered our radio buttons does not show selection when is no color setting. I'm investigating the widget library as well and noticed the same issue.

Is this a feature request (new component, new icon), a bug, or a general issue?
bug

Is this issue related to a specific component?
radio button

What did you expect to happen? What happened instead? What would you like to see changed?
The radio button should look selected.

What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

Steps to reproduce the issue
Set FF in color override setting.
Firefox Version: 60.1.0esr (64-bit)
Firefox > Tools > Options… > Content > Fonts & Colors: Colors… >
(check/uncheck) Use system colors
Override the colors specified by the page… = Always > OK > OK

View the radio buttons in the story book.

**Comment by tw15egan** [comment]
Do you have a suggestion on how to fix this? I am unfamiliar with the FF "no color" mode

**Comment by ninja511** [comment]
@aradhikaIbm

**Comment by aradhikaIbm** [comment]
I don't have any ideas yet. I've not had a chance to research it either.

**Comment by aradhikaIbm** [comment]
So I'm stilling investigating on my end. But another application seems to hide the image look and show the default input element. I'm asking to if someone knows how. 

![image](https://user-images.githubusercontent.com/22798731/46805425-01ade580-cd33-11e8-92c5-112b68014352.png)
The left side FF in no color mode and the right is chrome in normal mode.

**Comment by tw15egan** [comment]
Hmm, is there a css selector to target that situation?

**Comment by aradhikaIbm** [comment]
Not sure. Asking someone.

**Comment by dakahn** [comment]
@ninja511 is this still an issue, friend?

**Comment by ninja511** [comment]
@aradhikaIbm opened the initial issue. I just moved it to the correct repo

**Comment by aradhikaIbm** [comment]
I'm unsure of the current state of the problem. Our business priorities have shifted since opening the issue.

**Comment by joshblack** [comment]
Related: #3184

**Comment by tw15egan** [comment]
@dakahn any idea if this one was taken care of with https://github.com/carbon-design-system/carbon/pull/3442?

**Comment by dakahn** [comment]
Hey there! We don't support "no-color" mode as a means of high contrast. Happy to answer any further questions. Thanks for reporting.

**Comment by elizabethsjudd** [comment]
@dakahn can you add the lack of support for "no-color" to the HCM support documentation so we have a place to point our users to?
